app/gui/src/project-view/components/widgets/DropdownWidget.vue

```diff
@@ -63,6 +66,24 @@ const styleVars = computed(() => {
     '--extend-margin': `${0.2 / (graphNavigator?.scale ?? 1)}px`,
   }
 })
+
+/**
+ * Recursively find the nearest <li> element among ancestors of the event target.
+ * It is used to guarantee a stable target no matter the exact location of the click.
+ */
+function findTargetLiElement(target: EventTarget | null): HTMLElement | null {
+  if (!(target instanceof HTMLElement)) return null
+  let targetLiElement = target
+  while (!(targetLiElement instanceof HTMLLIElement)) {
+    targetLiElement = targetLiElement.parentElement as HTMLElement
+  }
+  return targetLiElement
+}
```

Comment:
Couldn't we just use `event.currentTarget` on `li` element?

If not, add explanation somewhere, why.

Reply:
Ah, yep, it will work.